data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetCommitteesTest.java

```diff
@@ -101,20 +86,21 @@ public void shouldReturnBadRequestWhenNoEpochIsSupplied() throws Exception {
   }
 
   @Test
-  public void shouldReturnEmptyListWhenAFutureEpochIsRequested() throws Exception {
```

Comment:
i'd still like a test to show behaviour on future epoch, and we definitely want that not cached

Reply:
Added `shouldHandleFutureEpoch` and `shouldHandleMissingFinalizedState`